src/app/containers/StoryPromo/index.jsx

```diff
@@ -174,13 +169,11 @@ const StoryPromoContainer = ({
 
   const headingTagOverride = getHeadingTagOverride({
     pageType,
-    isRecommendation,
     isContentTypeGuide,
   });
 
   const StyledHeadline = styled(Headline)`
     ${() =>
-      isRecommendation &&
```

Comment:
Ah, removing this means we are now applying the padding to every item, not just recommendations. This is why the snapshots are needing updated.